static/app/views/insights/uptime/components/percent.tsx

```diff
@@ -17,14 +17,16 @@ type UptimePercentProps = {
 };
 
 export function UptimePercent({summary, note, size}: UptimePercentProps) {
-  const knownChecks = summary.totalChecks - summary.missedWindowChecks;
+  // Do not consider missed or failed checks in the downtime % calculations
+  const knownChecks =
+    summary.totalChecks - summary.missedWindowChecks - summary.failedChecks;
```

Comment:
The main fix here, we don't include failed checks as part of the % uptime calculation